### ISSUE TITLE
Confusing search result hit for an http URL which redirects to an https ditto

### ISSUE THREAD

**Issue by petsva** [webrecorder/pywb#618]
## Problem

Note: I don't know if this is a bug, a configuration error or a missing feature. I would be grateful for any information on this.

### Situation
A warc file contains a response for an https URL of the entrance page of a site **and** a 302 response for the same URL except for "http:" instead of "https:". The latter response redirects to the former (https) URL. 

### Search
If you then search with pywb for the entrance page/site, you get hits for both responses in the search results, with different time stamps. And this is regardless of if you specify http, https or none of them in the search string. The links of both hits goes (despite being different URL:s) to a page showing the https page.

This is confusing for the user. There seems to be two captures of the same page, with just hours in between, and they have exactly the same content and apperance.

## Solution ideas

The redirected URL hit item in the list could either be inhibited or else it should be clearly stated that it is redirected. Or some other solution which reduces confusion.

## Additional context

Our test installation is out-of-the-box, probably all default. The warc file is made by Heritrix.

I've looked at issues here and searched in docs and source code but didn't find anything specifically solving this. But I've just started to get acquainted with pywb.

**Comment by edsu** [comment]
One solution to this that @ldko suggested on today's OH-SOS call was to follow practice in Internet Archive's Wayback of color coding captures in the calendar.

**Comment by petsva** [comment]
Yes, that would be good. But the colour difference is hard to see in the time stamps when you click, so some extra marking is needed. And also (from Slack:)
